addon-perf/index.html

```diff
@@ -12,13 +12,26 @@
     <script type="text/javascript" src="/lib/jquery.csv-0.8.2.js"></script>
     <link rel="stylesheet" href="addon-perf.css" />
     <link rel="stylesheet" href="http://code.jquery.com/ui/1.10.4/themes/smoothness/jquery-ui.css">
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-49796218-2"></script>
     <script>
-      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-      })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-      ga('create', 'UA-49796218-2', 'mozilla.org');
-      ga('send', 'pageview');
+      var _dntStatus = navigator.doNotTrack || navigator.msDoNotTrack;
```

Comment:
Does our GA setup not do DNT automatically?

Reply:
If we need our own DNT check, then it should go into a separate function for readability